### DISCUSSION TITLE
Difference between auto_stack=True to 'best_quality' preset

### DISCUSSION THREAD

**Discussion by borisRa** [autogluon/autogluon#1658] [Q&A] [answered]
Hi,

What is the difference between this two options :

```
1)predictor.fit(train_df , auto_stack=True) 
2)predictor.fit(train_df , presets='best_quality') 
```

Thanks !
Boris

**Comment by Innixma** [comment, marked as answer]
They are currently identical in v0.4 (and all prior versions), however future versions of AutoGluon could change what `best_quality` does such that it is more than `auto_stack=True`.
